### DISCUSSION TITLE
Trackpad Gestures

### DISCUSSION THREAD

**Discussion by eliasacab** [AppGround-io/bluetooth-keyboard-and-mouse-support#131] [Ideas]
I love your app! So reliable and useful. I'm currently using it for my iPad and I was wondering how hard it would be to implement 2/3 finger gestures that would work with iPadOS 13.4? The Magic Trackpad can be used as a reference. I'm willing to be a tester if needed. Thanks!

**Comment by zorazor** [comment]
Great idea!
Linux supports multitouch/gestures, too.

**Comment by ricardohnn** [comment]
Wow... Totally agree with the 3 finger gestures... I noticed that this app CAN work on the ipad... Would be amazing to have those 3 finger gestures.
